### ISSUE TITLE
Beacons to OGN aprs servers

### ISSUE THREAD

**Issue by Meisterschueler** [3s1d/fanet-stm32#4]
We can see these beacons from fanet devices:

```
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD -04fpm
FNBFF00BD>OGNFNT,TCPIP*,qAC,GLIDERN2:/182547h3820.72NI00028.78E&/A=000043
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD +03fpm
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD -01fpm
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD -03fpm
FNBFF00BD>OGNFNT,TCPIP*,qAC,GLIDERN2:/182547h3820.72NI00028.78E&/A=000043
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD +03fpm
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000042 !W74! id1FFF00BD +01fpm
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000042 !W74! id1FFF00BD -06fpm
FNBFF00BD>OGNFNT,TCPIP*,qAC,GLIDERN2:/182547h3820.72NI00028.78E&/A=000043
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD -01fpm
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD +01fpm
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD -00fpm
FNBFF00BD>OGNFNT,TCPIP*,qAC,GLIDERN2:/182547h3820.72NI00028.78E&/A=000042
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD +02fpm
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD +06fpm
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD +01fpm
FNBFF00BD>OGNFNT,TCPIP*,qAC,GLIDERN2:/182547h3820.72NI00028.78E&/A=000043
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD -04fpm
FNTFF00BD>OGNFNT,qAS,FNBFF00BD:/182547h3820.71N/00028.78Wg295/000/A=000043 !W74! id1FFF00BD -05fpm
```

My thoughts:

1. a device should send informations only once per timestamp. Multiple beacons with same timestamp and different climb rate seems invalid. One information per timestamp means: one beacon or beacons with different types (e.g. position beacon + status beacon as the tracker does).
2. we see "two devices". The device "FNB" is a "receiver" sending beacons to GLIDERN2, the device "FNT" is the aircraft and it is received by the "FNB". Both have the same dstcall. This behaviour is distantly related to the APRS beacons for <= 0.2.6. But: the "receiver" beacon is just a subset of the "aircraft" beacon.

My idea: the FNT behaviour should be the same as the naviter beacons:

- no more FNTxxxxxx>OGNFNT,qAS,FNBxxxxxx:/blabla since it is the same device
 ... but a FNTxxxxxx>OGNFNT,qAS,GLIDERN1:/blabla --> for beacons from the "device"
 ... and FNTyyyyyy>OGNFNT,FNTxxxxxx*,qAS,GLIDERN1:/blabla --> for device yyyyyy seen and relayed by the device xxxxxx
- no more FNB beacons since they are quite useless beside the FNT beacons (correct me if I am wrong)
 ... but a FNTxxxx>OGNFNT,qAS,GLIDERN1:>blabla --> status beacon with information about the skytraxx software or something like this

**Comment by 3s1d** [comment]
This is not an FANET related issue. The device in question does not even have an FANET model build in (see ID FFxxxx). It's a Skytraxx 3.0 w/ WiFi. Please tell me your mail address or contact me via my Skytraxx contact.

In short:
As during the time of development there was no way to add other devices than OGN/FLARM. The 'naviter interface' was not present back than. I tried to expend the interface but nobody from OGN reacted on my request. So I botched I'net connected devices and FANET devices in! Here the base station is only emulated as it is required to get displayed on glidertracker.de (w/o a base station they do not display anything). In case of a real FANET devices it is really a flying base station which will forward any other FANET node to the OGN servers. So it's mandatory in that cases. I'd be happy to change the protocol to a suitable modern one, but i won't do it every 6month (got other things to do).

During none-flight mode the position should be reported every 20sec and the base station gets updated every 60sec. So the 3 to 1 line ratio is correct. The constant time may be a GPS-fix related bug which I have to check.

Cheers
Juergen
